action.yaml

```diff
@@ -20,16 +20,24 @@ inputs:
     description: 'Path to source.extension.vs'
     required: false
     default: ''
+  version-number:
+    description: If provided, will be used directly as the version of the VSIX, and no incrementing will happen.
+    required: false
+    default: ''
 outputs:
   version-number:
     description: 'The resulting version number'
-    value: ${{ steps.version_increment.outputs.version }}
+    value: ${{ steps.version_set.outputs.version }}
 runs:
   using: "composite"
   steps:
-    - name: Increment VSIX version
-      id: version_increment
+    - name: Set VSIX version
+      id: version_set
```

Comment:
I'd like to not change the ID here -- some may be relying on the steps.id output in random places and that may be an unexpected breaking change...

Reply:
Hmm, can you access the output of a step within a composite action you call from the outside? That would be a strange breach of encapsulation. Anyway, I renamed it back.